CDCgov/prime-reportstream
# 7629 Added Analytics service to TrackAppInsightEvent

This PR ...

Test Steps:
1. Log into RS
2. Navigate to either Daily Data or Submission History
3. Select start and to dates and click Filter
4. App Insights should capture the event name ("Daily Data | Table Filter", "Submission History | Table Filter") along with the selected dates within the Logs or Usage Events

*If you are running locally you will need to set the following keys in your .env.development file:
REACT_APP_APPINSIGHTS_KEY
REACT_APP_APPLICATIONINSIGHTS_CONNECTION_STRING

## Changes
- Added Analytics service to TrackAppInsightEvent.
- Updated TableFilters.tsx to track event when filtering.
- Created enum to store Feature types

## Checklist

### Testing
